app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(argument: str) -> tuple:
+    if len(argument.rsplit(", ")) == 3:
+        cp, first_file, second_file = argument.split(", ")
+        if first_file != second_file and cp == "Copied":
+            with open(first_file, "rb") as first, (
+                    open(second_file, "wb")
+            ) as dst:
+                dst.write(first.read())
+            if open(first_file).read() == open(second_file).read():
+                print("Copied")
+                return
+
+argument1 = "Copied, txt.txt, ff929.txt"
```

Comment:
<img width="576" alt="image" src="https://github.com/mate-academy/py-copy-file/assets/65166203/186ac357-c385-49d9-9f3d-7e305dabe465">
First of all command must be space separated due to task, not comma + space
Also you must check that command is cp